### PR TITLE
Changed system console schema to use full key for settings

### PR DESCRIPTION
**Pull request by hmhealey**
This allows us to mix settings from different sections of the config on a single system console page and it'll also be used for https://mattermost.atlassian.net/browse/MM-9849.

@jwilander Are there tests in place for plugin settings or can you point me to a test plugin that I can use that has system console settings? I want to make sure I didn't break anything with those by making this change

### CODE REVIEW

**Comment by jwilander** [comment]
There is `tests/components/admin_console/custom_plugin_settings.test.jsx` which seem to be passing

**Comment by hmhealey** [comment]
I've switched it to Elias so we can get it in on time. This is mostly Jesús's code, so I'll still grab him for a followup review once he's back

File: components/admin_console/admin_definition.jsx [hunk 1]
```diff
@@ -521,7 +521,7 @@ export default {
                         },
                         {
                             type: Constants.SettingsTypes.TYPE_BOOL,
-                            key: 'EnableDiagnostics',
+                            key: 'LogSettings.EnableDiagnostics',
```
Comment (enahum) [inline components/admin_console/admin_definition.jsx]: Im not seeing the settings for the json format for the logs, are they going to be included?

Comment (hmhealey) [reply components/admin_console/admin_definition.jsx]: Only if whoever added that setting wants to add it. This PR isn't adding any new settings. It's just changing how the existing ones are defined so I can add more in my other PR

**Pull request merged**